### ISSUE TITLE
Tranning issue, Need guidance

### ISSUE THREAD

**Issue by mujtabaAli96** [fizyr/keras-retinanet#1567]
I have 250k labels in CSV and using Colab pro to train my model. currently, I am training my model in 20k chunks with Epoch=10, 
steps =20k, and batch size =1 when the training breaks in-between I put the last epoche file name in the screenshot and start it again
( e.g. if its break after the 6th epoch I put the 6th  snapshot file name in the snapshot and train it again till next 4 epoch) and once a chunk completed I put the 10th epoch file name in the Weights (without converting it ) update the labels file with next 20k labels and repeat the process.
but not getting the best result, especially every new (chunk) model decrease the number of detected objects.
Waiting for your guidance 
Thanks
